client/components/Post.jsx

```diff
@@ -1,36 +1,36 @@
 import styles from './Post.module.css';
+import PropTypes from 'prop-types';
 
 const Post = ({ profileImg, profileName, postContent, upVote, downVote, comment, timeStamp }) => {
 
-    // for place holder purposes only
-    const data = {
-        profileImg: "https://cdn-icons-png.flaticon.com/512/3899/3899618.png",
-        profileName: "Profile Name",
-        postContent: "Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur sunt quibusdam nam illo eveniet aliquid?",
-        upVote: "Up Vote",
-        downVote: "Down Vote",
-        comment: "Comment",
-        timeStamp: "18:44 11/12/2023"
-    }
+    Post.propTypes = {
```

Comment:
that's a nice addition, but I wouldn't say it is required at the moment.
besides, in the future you'll use typescript which is better in my opinion.
your choice to keep it, just keep a watch out for type warnings